### PR TITLE
User story 21

### PR DESCRIPTION
**Pull request by Rostammahabadi**
Created method for updating password
updated view with the format for entering in the password and confirming password

### CODE REVIEW

File: app/controllers/default_user/profile_controller.rb [hunk 1]
```diff
@@ -4,14 +4,26 @@ def index
     end
 
     def edit
+      if params[:pw] == "change"
+        @var = 1
```
Comment (melatran) [inline app/controllers/default_user/profile_controller.rb:8]: Can someone explain what this @var = 1 means?

File: app/controllers/default_user/profile_controller.rb [hunk 2]
```diff
@@ -4,14 +4,26 @@ def index
...
+      end
     end
 
     def update
```
Comment (melatran) [inline app/controllers/default_user/profile_controller.rb:12]: Nice method to take in different edits of a profile

**Comment by melatran** [comment]
This looks great. Nice robust tests

File: app/views/default_user/profile/edit.html.erb [hunk 3]
```diff
@@ -20,15 +20,14 @@
 
     <%= submit_tag "Submit update" %>
```
Comment (JPuchalla) [inline app/views/default_user/profile/edit.html.erb:21]: Great initial refactor on this.

**Pull request merged**